<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Генерация палитры</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QPushButton" name="selectButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>251</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Выбрать изображение</string>
       </property>
      </widget>
      <widget class="QPushButton" name="saveButton">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>20</y>
         <width>221</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>сохранить палитру</string>
       </property>
      </widget>
      <widget class="QPushButton" name="exitButton">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>20</y>
         <width>211</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>выйти</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>961</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimum">
        <number>3</number>
       </property>
       <property name="maximum">
        <number>15</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBothSides</enum>
       </property>
       <property name="tickInterval">
        <number>9</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>291</width>
         <height>51</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>203</red>
             <green>203</green>
             <blue>217</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>45</red>
             <green>134</green>
             <blue>217</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>22</red>
             <green>14</green>
             <blue>14</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>239</red>
             <green>239</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>45</red>
             <green>134</green>
             <blue>217</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>203</red>
             <green>203</green>
             <blue>217</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>45</red>
             <green>134</green>
             <blue>217</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>203</red>
             <green>203</green>
             <blue>217</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <strikeout>false</strikeout>
         <stylestrategy>PreferDefault</stylestrategy>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Количество цветов:</string>
       </property>
      </widget>
      <widget class="QLabel" name="numbers">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>130</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>203</red>
             <green>203</green>
             <blue>217</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>45</red>
             <green>134</green>
             <blue>217</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>22</red>
             <green>14</green>
             <blue>14</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>239</red>
             <green>239</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>45</red>
             <green>134</green>
             <blue>217</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>203</red>
             <green>203</green>
             <blue>217</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>45</red>
             <green>134</green>
             <blue>217</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>203</red>
             <green>203</green>
             <blue>217</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <strikeout>false</strikeout>
         <stylestrategy>PreferDefault</stylestrategy>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QLabel" name="imagelbl">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>200</y>
         <width>801</width>
         <height>351</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="palettelbl">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>130</y>
         <width>661</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="loadButton">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>241</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Загрузить палитру</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
